setup.cfg

```diff
@@ -3,3 +3,12 @@ description-file = README.rst
 
 [bdist_wheel]
 universal=1
+
+[tool:pytest]
+norecursedirs= .tox
+
+[flake8]
+# E251 unexpected spaces around keyword / parameter equals
+# W391 blank line at end of file
+# E501 line too long (> 79 characters)
+ignore = E501,W391,E251
```

Comment:
Why ignore these 3? Also, instead of ignoring E501, can we configure the max line length instead?